Resources/Prototypes/Recipes/Reactions/pyrotechnic.yml

```diff
@@ -37,12 +37,14 @@
       amount: 3
   effects:
   # TODO electro's pretty explosions PR make this big and firey!!
+  # Welp its fiery now (low damage). Someone else needs to come in and define how big it should get & how it compares to potassium
   # TODO solution temperature!!
   - !type:ExplosionReactionEffect
-    devastationRange: 0
-    heavyImpactRange: 0
-    lightImpactRange: 2
-    scaled: false
+    explosionType: Default # 15 damage per intensity
+    maxIntensity: 2 # at most 30 damage per tile.
+    intensityPerUnit: 2 # 80 total input reagent units for max size
```

Comment:
clf3splosions are intended to be non-scaled, though maybe it should scale just really slowly (something like maxing out at 5-10u). ideally this is just a 3x3 fireball with very little damage, mostly intended for pyrotechnics.

Reply:
I tweaked the values, it should now only ever deal 15 damage at most, and maxes out at 12 total reagents with this area:
![clf3](https://user-images.githubusercontent.com/60421075/159149897-b607c866-1f80-465a-8d7e-cc57d4da5ca3.png)
Feel free to also just weak them yourself.

Reply:
sounds good